### PR TITLE
Switch to plugin-sdk on production

### PR DESCRIPTION
**Pull request by thelink2012**
...plus a fix (#1)

### CODE REVIEW

**Comment by thelink2012** [comment]
Once on master, we can update the solution or so and add the submodule.

File: CTheScripts.h [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    inline	short		ReadDataArrayIndex() {
+        return ReadDataWord();
+    }
+    inline	short		ReadDataArraySize() {
+        return ReadDataByte();
+    }
+    inline	short		ReadDataArrayFlags() {
+        return ReadDataByte();
+    }
+    /*
```
Comment (x87) [inline CTheScripts.h]: Do we need this commented code?

**Comment by thelink2012** [comment]
So...

**Comment by x87** [comment]
@thelink2012 @Deji69 
Since Deji haven't replied, let's assume he's ok with changes. @Deji69 if you have any objections, please say anyway.

**Pull request merged**